sherlock/resources/data.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Sherlock Targets",
+    "description": "Social media target to probe for existence of usernames",
+    "type": "object",
+    "patternProperties": {
+        "^(?!\\$).*?$": {
+            "type": "object",
+            "required": [ "url", "urlMain", "errorType", "username_claimed" ],
+            "properties": {
+                "url": { "type": "string" },
+                "urlMain": { "type": "string" },
+                "urlProbe": { "type": "string" },
+                "username_claimed": { "type": "string" },
+                "regexCheck": { "type": "string" },
+                "isNSFW": { "type": "boolean" },
+                "headers": { "type": "object" },
+                "request_payload": { "type": "object" },
+                "tags": {
+                    "oneOf": [
+                        {
+                            "type": "string",
+                            "enum": [ "adult", "gaming" ]
+                        },
+                        {
+                            "type": "array",
+                            "items": {
+                                "type": "string",
+                                "enum": [ "adult", "gaming" ]
+                            }
+                        }
+                    ]
+                },
```

Comment:
What do you think of having the array type? This way, when adding new tags, it will only have to be added once

Reply:
By having both types in a `oneOf`, we're affording the user the ability to use either format depending on the needs of the target. It functions similarly to how errorMsg functions now (allowing either a string or a list of strings), and the primary difference to the errorMsg definition here is just that errorMsg allows arbitrary strings and tags requires one of a predefined list of strings.

So...

```json
"tags": "gaming"
"tags": [ "adult", "gaming" ]
```

are both valid, but only one variation can be entered at a time.